extensions/postinstall.mjs

```diff
@@ -26,7 +26,10 @@ function processRoot() {
 function processLib() {
 	const toDelete = new Set([
 		'tsc.js',
+		'_tsc.js',
+
 		'typescriptServices.js',
+		'_typescriptServices.js',
```

Comment:
FWIW `typescriptServices` hasn't existed since TS 4.9; I don't think you're running with older versions of TS so this should be safe to drop (and `_typescriptServices` has never existed).